build.gradle

```diff
@@ -21,36 +14,47 @@ configurations {
 
 allprojects {
   group = "com.linkedin.coral"
+  apply plugin: "com.diffplug.spotless"
+
   repositories {
     mavenCentral()
     maven {
       url 'https://linkedin.bintray.com/maven/'
     }
   }
-}
 
-subprojects {
-  apply plugin: "com.github.hierynomus.license"
-
-  license {
-    header = file("${rootDir}/gradle/license/LICENSE_HEADER")
-    ext.year = Calendar.getInstance().get(Calendar.YEAR)
-    skipExistingHeaders true
-    ignoreFailures false
-    strictCheck true
-
-    exclude '**/*.avsc'
-    exclude '**/*.json'
+  spotless {
+    groovyGradle {
```

Comment:
Can we also use `licenseHeaderFile` for groovy sources? https://github.com/diffplug/spotless/tree/main/plugin-gradle#groovy

Reply:
Ok, seems like can't be supported for Gradle files easily because there is no generic prefix for `.gradle` files. It uses `package` statement to identity the start of `.groovy` and `.java` files.

Reply:
Yup, that's the problem I ran into. Disappointing to say the least.